src/app/pages/sharing/iscsi/target/all-targets/target-details/target-details.component.ts

```diff
@@ -32,6 +35,7 @@ export class TargetDetailsComponent {
   readonly target = input.required<IscsiTarget>();
 
   targetPort = signal<FibreChannelPort>(null);
+  connections = toSignal(this.api.call('fcport.status'));
```

Comment:
there probably may be same connections? Maybe we can insure that we only show unique connections?

I did it for iSCSI sessions:

```
.subscribe((sessions) => {
        // Remove duplicates using lodash's uniqBy
        const uniqueSessions = uniqBy(sessions, (session) => `${session.initiator}_${session.initiator_addr}`);
        this.sessions.set(uniqueSessions);
      });
```

Reply:
This is not a case. Here sessions transformed into boolean. Anyway thanks for the tip.